copier.yml

```diff
@@ -296,8 +302,12 @@ domains_test:
     - hosts: The domain names. List of strings.
     - path_prefixes: The path prefixes (if any) that will match this routing rule.
       List of strings. Default: [].
-    - redirect_to: Apply a 301 redirection to this other domain.
+    - redirect_to: Apply a redirection to this other domain. By default, it will be a
+      302 redirection, but it can be turned into a 301 redirection by enabling the
+      `redirect_301` option.
       String. Default: null.
+    - redirect_301: Return 301 instead of 302 in a Traefik redirection. Boolean.
```

Comment:
I'd say `redirect_permanent` is more understandable.

Also IMHO this should be true by default, as a proxy redirection is gonna be almost always a 301. However that could introduce some backwards incompatibility. Nothing that can't be fixed with a major release bump.